src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,33 @@
+.executive-education-2u-error-illustration {
+    align-items: center;
+    margin-bottom: 32px;
+}
+
+.executive-education-2u-error-heading {
+    align-items: center;
+    line-height: 60px;
+    margin-bottom: 32px;
```

Comment:
nit: use `.mb-4.5` to avoid defining custom styles for spacing ([docs](https://paragon-openedx.netlify.app/foundations/spacing/)).